kitsune/users/management/commands/purge_hashes.py

```diff
@@ -1,19 +0,0 @@
-from datetime import datetime, timedelta
```

Comment:
Similarly, in scripts/cron.py#job_purge_hashes, same comment as above wrt DMS.

I haven't gotten there yet, but do we have a migration to remove the user.password column from the DB?

Reply:
It's a no about the migration.